LICENSE

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
Why GPL?

Reply:
Nevermind, checked here and it is LGPL

> The main difference between the GPL and the LGPL is that the latter allows the work to be linked with (in the case of a library, "used by") a non-(L)GPLed program, regardless of whether it is free software or proprietary software.[2] The non-(L)GPLed program can then be distributed under any terms if it is not a derivative work.

Seems fair

Reply:
that's what @mjgarton told me to use last time 😅